src/plugins/intel_cpu/src/emitters/snippets/x64/jit_brgemm_emitter.cpp

```diff
@@ -66,17 +66,20 @@ std::set<std::vector<element::Type>> jit_brgemm_emitter::get_supported_precision
     using brgemm_utils::BRGEMM_TYPE;
     if (brgemm->get_type() == BRGEMM_TYPE::STAND_ALONE) {
         return {{element::f32, element::f32}};
-    } else if (brgemm->get_type() == BRGEMM_TYPE::REPACKING_ONLY) {
+    }
+    if (brgemm->get_type() == BRGEMM_TYPE::REPACKING_ONLY) {
```

Comment:
seems better to rewrite with switch case

Reply:
Done